dependent/journal-of-hip-preserving-surgery.csl

```diff
@@ -0,0 +1,14 @@
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <info>
+    <title>Journal of Hip Preserving Surgery</title>
```

Comment:
@zuphilip, the title seems to be "Journal of Hip Preservation Surgery" (not "preserving")